src/sys/context.rs

```diff
@@ -0,0 +1,25 @@
+use libc;
```

Comment:
convention thusfar is using the header name, in which case this should be at `src/ucontext.rs`. There is #214 to discuss how to handle it though...